packages/core/test/Asset/util/metadatumToCip25.test.ts

```diff
@@ -191,7 +191,16 @@ describe('NftMetadata/metadatumToCip25', () => {
     expect(metadatumToCip25(asset, metadatum, logger)).toEqual(minimalConvertedMetadata);
   });
 
-  it('supports image with ipfs protocol', () => {
+  it('supports image with ipfs protocol as string', () => {
+    const metadatum = createMetadatumWithFiles(
+      [],
+      ['ipfs://bafybeihtdkq3ntfcewytdaimnrslpxsatsg47e3bqlsgi', '3jkax65pypymi']
+    );
+    const result = metadatumToCip25(validAsset, metadatum, logger);
+    expect(result?.image).toEqual(['ipfs://bafybeihtdkq3ntfcewytdaimnrslpxsatsg47e3bqlsgi3jkax65pypymi']);
```

Comment:
There's no use of having this as an array, let's update the type for `image` and for `files`
```suggestion
    expect(result?.image).toEqual('ipfs://bafybeihtdkq3ntfcewytdaimnrslpxsatsg47e3bqlsgi3jkax65pypymi');
```

Reply:
bfc177bd8b2c90e2256a12d122edd4d506ca436d